frontend/packages/dev-console/integration-tests/features/Helm/helm-compatability-ODC-3430.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Helm Chart
+    User will be able to update the chart versions or values config of a helm release
+
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke, @manual
+Scenario: Compatible helm charts
+    Given user is at +Add page
```

Comment:
```suggestion
    Given the user is on the +Add page
```

Reply:
In every feature file, it is written as the 'user at +Add page'. If I changed it here, I need to change it everywhere or need to write different step definition for this only one statement.